proposals/stringref/Overview.md

```diff
@@ -493,11 +518,10 @@ wtf8_policy ::= 0x00 ⇒ utf8
              |  0x02 ⇒ replace
 
 instr ::= ...
-       |  0xfb 0x80 $mem:u32              ⇒ string.new_wtf8 $mem
+       |  0xfb 0x80 $mem:u32 $policy:u32  ⇒ string.new_wtf8 $mem $policy
```

Comment:
We could limit the `$policy` immediate to one byte as `u8`. Probably doesn't make a difference in practice.